package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "build.rollup": "npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
-    "build": "npm run clean && npm run tsc.scripts && npm run tsc.prod && node scripts/build/esbuild/build.js --prod --ci",
+    "build": "npm run clean && npm run tsc.scripts && npm run tsc.prod && node scripts/build --prod --ci",
```

Comment:
since `scripts/index.ts` now imports from `scripts/esbuild/build.ts` we run that here (`scripts/build` has the output of building the `scripts/` directory, so `scripts/build/index.js` is what's going to be run here, which is compiled from `scripts/index.ts`)